data/visibility.txt

```diff
@@ -533,11 +533,11 @@ world  00000000000000000000  +
         fort  00000000000011111111  -
         gallows  00000000000000001111  -
         memorial  00000000000000011111  +
-            cross  00000000000000011111  -
-            plaque  00000000000000011111  -
+            cross  00000000000000000111  -
+            plaque  00000000000000000000  -
             sculpture  00000000000000011111  -
             statue  00000000000000011111  -
-            stolperstein  00000000000000011111  -
+            stolperstein  00000000000000000000  -
```

Comment:
?

Reply:
Those types were "displayed" using a zero-icon in vehicle style (so were invisible actually). Now they're removed from there properly.